src/components/routing/AppRouting/index.js

```diff
@@ -2,17 +2,21 @@ import React from 'react';
 import { Switch, Route } from 'react-router-dom';
 import { AddItemPage } from '../../../pages/AddItemPage';
 import { ListPage } from '../../../pages/ListPage';
+import { Welcome } from '../../../pages/Welcome';
 
 export function AppRouting() {
   return (
     <div>
       <Switch>
-        <Route exact path="/list">
+        <Route path="/list">
           <ListPage />
         </Route>
         <Route path="/addItem">
           <AddItemPage />
         </Route>
+        <Route path="/">
+          <Welcome />
+        </Route>
```

Comment:
```suggestion
       <Route exact path="/">
          <Welcome />
        </Route>
        <Route path="/list">
          <ListPage />
        </Route>
        <Route path="/addItem">
          <AddItemPage />
        </Route>
```

I make this suggestion only as a semantic improvement, since `path='/'` is the start of our application and it would make sense for it to be the first route declared.